wiki/Tournaments/4DM/2/en.md

```diff
@@ -8,7 +8,7 @@ tags:
 
 ![4DM2 logo](img/logo.jpg)
 
-The **4 Digit osu!mania World Cup 2** (***4DM2***) is a worldwide country-based double-elimination three-on-three osu!mania 4K tournament hosted by ![][flag_US] [Logan636](https://osu.ppy.sh/users/9423766) and ![][flag_CA] [Sinaeb](https://osu.ppy.sh/users/1576095). This is the second installment of the 4 Digit osu!mania World Cup.
+The **4 Digit osu!mania World Cup 2** (***4DM2***) is a worldwide country-based double-elimination three-on-three osu!mania 4K tournament hosted by ![][flag_US] [Logan636](https://osu.ppy.sh/users/9423766) and [Sinaeb](https://osu.ppy.sh/users/1576095). This is the second installment of the 4 Digit osu!mania World Cup.
```

Comment:
was this intentional? happens elsewhere for same user

Reply:
Unintentional, i botched the find and replace after adding participants.